qualityattenuation/qualityattenuation.go

```diff
@@ -0,0 +1,133 @@
+// Implements a data structure for quality attenuation.
+
+package qualityattenuation
+
+import (
+	"math"
+
+	"github.com/influxdata/tdigest"
+)
+
+type SimpleQualityAttenuation struct {
+	EmpiricalDistribution     *tdigest.TDigest
+	Offset                    float64
+	Offset_sum                float64
+	Offset_sum_of_squares     float64
```

Comment:
If you don't mind, can we convert to `camelCase` (what go calls `MixedCase`? I prefer `snake_case` but go habits are for the former. (https://go.dev/doc/effective_go#mixed-caps)